.github/workflows/release-candidate.yaml

```diff
@@ -79,7 +76,7 @@ jobs:
         run: |
           cd registry
           mvn -N io.takari:maven:wrapper -Dmaven=3.6.3
-          ./mvnw clean install -Pprod -Psql -Pkafkasql -DskipTests -Pmultitenancy -Dmaven.wagon.httpconnectionManager.maxTotal=30 -Dmaven.wagon.http.retryHandler.count=5
+          make SKIP_TESTS=true BUILD_FLAGS='-Dmaven.wagon.httpconnectionManager.maxTotal=30 -Dmaven.wagon.http.retryHandler.count=5' build-all
```

Comment:
The build commands are specified in the makefile. the benefit of doing this is that whenever there is a change in the build command, instead of making changes in multiple places, we can simply change the build command in Makefile. We no longer have to search for all the places where the build command is specified in the workflow and change them one by one. 

If you like the idea, we can probably use similar strategies for all the workflows. Currently i have only done the changes in the verify and release workflow.